### PR TITLE
Fixes #68026: revise some errors about golint in audit packages #68113

### PR DESCRIPTION
**Pull request by mourya007**
What this PR does / why we need it:

According to this issue: #68026, I revise some errors about golint in audit ptackages as below

 staging/src/k8s.io/apiserver/pkg/audit
 staging/src/k8s.io/apiserver/pkg/audit/policy

Which issue(s) this PR fixes :

Ref #68026

### CODE REVIEW

File: staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go [hunk 1]
```diff
@@ -63,8 +64,9 @@ func LoadPolicyFromFile(filePath string) (*auditinternal.Policy, error) {
 	return ret, nil
 }
 
+// LoadPolicyFromBytes will  return the policy from the specified bytes
 func LoadPolicyFromBytes(policyDef []byte) (*auditinternal.Policy, error) {
-	policy := &auditinternal.Policy{}
+	poicy := &auditinternal.Policy{}
```
Comment (wenjiaswe) [inline staging/src/k8s.io/apiserver/pkg/audit/policy/reader.go]: This looks like a typo to me?

**Review by wenjiaswe — CHANGES_REQUESTED** [review]

**Comment by riaan53** [comment]
Hi @mourya007 , I noticed that there are some spaces in the middle of some of your comments and that some ends with a `.` and some dont. It looks like the standard is without `.`

**Pull request closed without merging**